zone/main.cpp

```diff
@@ -316,6 +316,11 @@ int main(int argc, char** argv) {
 		LogError("Loading npcs faction lists failed!");
 		return 1;
 	}
+	LogInfo("Loading splash faction hits");
+	if (!database.LoadSplashFaction(hotfix_name)) {
```

Comment:
Same here with `content_db`, though maybe there's something with shared memory I don't understand because I see others using `database` that are listed as content tables too